vagrant/provision/rds01/Configure-RDS.ps1

```diff
@@ -0,0 +1,36 @@
+# Parameters Hard Coded here because this needs to be run inside host.
+param(
+    $RDHostName = "rds01.lab.milliondollar.me.uk",
+    $RDCollectionName = "MillionDollarCollection"
```

Comment:
Both of these values can be passed in from your Vagrantfile.
See how I've done this in the other scripts, eg:
`machine.vm.provision "shell", path: "Vagrant/provision/all/Join-Domain.ps1", args: [domain_name, domain_admin_un, domain_admin_pw, dc01_ip]`

For each variable you need to pass in, you will need to create a new var at the top of Vagrantfile.
Nothing wrong with having defaults though :)

Reply:
Ah yes - I thought you'd pick up on this. This script is actually not useable through Vagrant  provisioning thanks to the double-hop WinRM issue. So I've just hard-coded those variables as I'll need to call the script from within the VM anyway. I had originally passed them in, until I hit this issue.

Reply:
For clarity - RD tries to use WinRM on the local machine you're trying to set it up on - which spins the whole remote provisioning out.

Reply:
Ah ok. Something to workout in a future version perhaps